developers/verifiers/multicredentials.md

```diff
@@ -20,34 +20,70 @@ Here's how to get a demo Veres Wallet if you need one:
 
 3. Once you click "Register", you'll receive an "Allow Wallet" modal. Make sure to click "Allow" on that modal, so the wallet can handle credentials on your behalf.
 
-Once that's done, proceed to VC Issuance.
+Once that's done, proceed to Verifiable Credential Issuance.
 
-## VC Issuance
+## Verifiable Credential Issuance
 
-1. Visit the Issuer section of the CHAPI Playground by visiting <https://playground.chapi.io/issuer>.
+First, visit [the Issuer section of the playground](https://playground.chapi.io/issuer).
 
-2. Select the "Alumni" icon in the top "Select a credential to issue" section.
+Next, select the "JFF x vc-edu PF3" icon in the top "Select a credential to issue" section.
 
-3. Click "Issue Verifiable Credential" to proceed to the (optional) DID Authentication screen.
+The credential JSON below is similar to what will be passed to the issuer to generate the credential.
 
-4. Once you've completed DID Authentication (or not), you'll be provided with the Verifiable Credential JSON. Click "Store in Wallet" to proceed.
+{% include example-preissuance-credential.html %}
 
-5. You'll land at the main CHAPI Playground screen. Click "Issuer Demo".
+Now, lets issue the credential. Click "Issue Verifiable Credential" to proceed to the (optional) DID Authentication screen.
```

Comment:
FYI I think we are planning on moving this DID Auth screen into a wallet connect button in the header as I was showing you the other day. Plans might have changed though I'm not sure

Reply:
Looking forward to that change.